src/utils/logger.js

```diff
@@ -1,42 +1,42 @@
-const { bgBlue, black, green } = require("chalk");
+const { bgBlue, black, green } = require("chalk")
 
 // makes logs look pretty and uniform
 
 function dateTimePad(value, digits) {
-    let number = value;
-    while (number.toString().length < digits) {
-        number = "0" + number;
-    }
-    return number;
+	let number = value
+	while (number.toString().length < digits) {
+		number = "0" + number
+	}
+	return number
 }
 
 function format(tDate) {
-    return (tDate.getFullYear() + "-" +
+	return (tDate.getFullYear() + "-" +
         dateTimePad((tDate.getMonth() + 1), 2) + "-" +
         dateTimePad(tDate.getDate(), 2) + " " +
         dateTimePad(tDate.getHours(), 2) + ":" +
         dateTimePad(tDate.getMinutes(), 2) + ":" +
         dateTimePad(tDate.getSeconds(), 2) + "." +
-        dateTimePad(tDate.getMilliseconds(), 3));
+        dateTimePad(tDate.getMilliseconds(), 3))
 }
```

Comment:
I would recomend the module [dateformat](https://www.npmjs.com/package/dateformat).
It has 0 dependencies and does all the heavy lifting.
`dateFormat(now, "mm-dd HH:MM:SS.l");` -> `05-13 03:37:20.000`